atoma-vllm/src/validation.rs

```diff
@@ -87,6 +87,10 @@ impl Validation {
             encoding.len()
         };
 
+        if input_len == 0 {
+            // TODO: handle the case in which input length == 0
```

Comment:
I am not certain what the behaviour of the system should be in the case of `input_len == 0`.. A model can still generate a response in this case, even though it is likely not what the user intended (this has a cost, right?).. Therefore, it might just be more appropriate for the node to simply commit to an empty request, instead.